src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/eventDetailsBuilder.hpp

```diff
@@ -153,6 +157,11 @@ class TEventDetailsBuilder final : public AbstractHandler<std::shared_ptr<TScanC
                 populateField(package, "/name"_json_pointer, osFullName);
                 populateField(package, "/type"_json_pointer, osType);
                 populateField(package, "/version"_json_pointer, osVersion);
+
+                // Fill the remaining fields with empty strings.
```

Comment:
These populated with EMPTY_FIELD are unnecessary, check line 63 of this file.

Reply:
Removed!